src/shared/interfaces/categorized-validation-errors.ts

```diff
@@ -0,0 +1,6 @@
+import { SchemaValidationErrors } from './schema-validation-errors';
+
+export interface CategorizedValidationErrors {
+  Errors: SchemaValidationErrors;
```

Comment:
Since this can be internal, external and errors or warnings a better name would be a more general `ValidationMessages`